src/main/java/gov/usgs/wma/waterdata/collections/CollectionsController.java

```diff
@@ -103,14 +107,16 @@ public String getOgcCollection(@RequestParam(value = "f", required = false, defa
 			},
 			externalDocs=@ExternalDocumentation(url="http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#_feature_")
 		)
+	@Parameter(name="bbox", in=ParameterIn.QUERY, description=BBOX_DESCRIPTION)
 	@GetMapping(value = "collections/{collectionId}/items", produces = MediaType.APPLICATION_JSON_VALUE)
 	public String getOgcCollectionFeatures(
 			@RequestParam(value = "f", required = false, defaultValue = "json") String mimeType,
 			@Min(value=1, message = LIMIT_VALIDATE_MESS) @RequestParam(value = "limit", required = false, defaultValue = "10000") int limit,
 			@Min(value=0, message = START_INDEX_VALIDATE_MESS) @RequestParam(value = "startIndex", required = false, defaultValue = "0") int startIndex,
-			@BBox @RequestParam(value = "bbox", required = false) BoundingBox bbox,
+			@BBox @RequestParam(value = "bbox", required = false) String bbox,
 			@PathVariable(value = PARAM_COLLECTION_ID) String collectionId, HttpServletResponse response) {
 
+		BoundingBox boundingBox = bbox == null ? null : new BoundingBox(bbox);
```

Comment:
Why did you stop letting Spring bind the BoundingBox? This seems like extra work and it is unclear to me if it actually works.

Reply:
When the param was the type was BoudningBox, the swagger UI showed the parameter
type as 'object' and wanted the user to enter a json object with fields: single, north,
south ..

![image](https://user-images.githubusercontent.com/54034802/78257553-9ecc8300-74bf-11ea-8064-2a60221cb0d2.png)

Reply:
Well, not sure why I did not see this before. Looks like there is a Straight forward
way to to update the swagger documentation using @Parameter
I'll revert the param back to a BoundingBox

Reply:
you might also be able to suppress the "single" with a jackson or openapi annotation and keep the other values.

Reply:
I think the suppress "single" was done in the last commit.
The last commit deleted this line and added Schema annotation to the openapi Parameter
annotation. It appears to have the desired effect of replacing the "object" type displayed 
in swagger ui with 'string' and allowing the user to enter the comma delimited string.